WebAPI/Controllers/UserController.cs

```diff
@@ -200,8 +200,24 @@ public async Task<IActionResult> LoginLdap([FromBody] LoginModel body)
                 ldapInformations = _userService.GetUserFromLDAP(body.Email, body.Password, _logger);
             } catch (LdapException e)
             {
-                _logger.LogError($"Exception while logging into ldap: {e}");
-                return ErrorResponse(e.ResultCode == 49 ? $"Meno alebo heslo nie je správne, skúste znova prosím." : $"Prepáčte, niečo na serveri nie je v poriadku, skúste neskôr prosím."); //49 = InvalidCredentials
+                var old_mail_format = body.Email + "@fri.uniza.sk";
+                User old_user = await _userService.GetUserByEmailAsync(old_mail_format);
+                if (old_user != null) {
+                    _logger.LogError($"User {body.Email} try to use FRI LDAP login.");
+                    var result = await _userService.ChangeUserEmail(old_user, body.Email + "@stud.uniza.sk");
```

Comment:
@stud.... to configuration or at least as some const atribute of the User (e.g. DefaultUserEmailDomain)